Source/d_main.c

```diff
@@ -1755,6 +1752,9 @@ void D_DoomMain(void)
       states[i].flags |= STATEF_SKILL5FAST;
   }
 
+  // killough 10/98: process all command-line DEH's first
```

Comment:
Comment, please.

Reply:
I'm thinking, maybe we should move D_ProcessDehCommandLine later, like Choco/Pr+? Here is the old Pr+ bugfix: https://sourceforge.net/p/prboom-plus/bugs/18/

Also, maybe refactor mobjinfo initialization into function and move it somewhere too.

Reply:
Right, let's call these functions in the same order as Pr+/Choco do (with comments where due). It doesn't make sense to maintain a different ordering just because MBF did so back then, DEH was a very different thing in 1999 than it is today.

Yes, let's please put the init stuff in a separate function and call it as soon as possible!